documentation/index.markdown

```diff
@@ -26,6 +26,8 @@ Puppet Server is the next-generation application for managing Puppet agents.
 * **Administrative API**
     * [Environment Cache](./admin-api/v1/environment-cache.markdown)
     * [JRuby Pool](./admin-api/v1/jruby-pool.markdown)
+* **Puppet API**
+    * [Environment Classes](./puppet/v3/environment_classes.markdown)
```

Comment:
Thanks, @gguillotte.  Moved the new docs under `./puppet-api/v3` in 2bc4c6e and also renamed `environment_classes.markdown` to `environment_classes.md`.